crates/circuit/src/dag_circuit.rs

```diff
@@ -1343,6 +1343,36 @@ impl DAGCircuit {
         self.copy_empty_like_with_capacity(0, 0, vars_mode)
     }
 
+    /// Put ``self`` into the canonical physical form, with the given number of qubits.
+    ///
+    /// This acts in place and is cheap.  It is intended for use when the DAG is known to already
```

Comment:
What is "cheap" in this context? I know what you mean, but it might be worth being a bit more explicit that you mean it's efficient/fast.